packages/ui/src/components/Question/Question.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { AccordionButton, AccordionIcon, AccordionItem, AccordionPanel } from '@chakra-ui/react';
+
+import { Typography } from '../Typography';
+
+export type QuestionProps = {
+  title: string;
+  content: string;
+};
+
+export const Question = ({ title, content }: QuestionProps) => {
+  return (
+    <AccordionItem borderColor="gray.200">
```

Comment:
🤔 Paddings for the question don't match mockup ones. There should be no horizontal padding and vertical padding should be equal to 24px on each side